code/datums/quirks/negative_quirks.dm

```diff
@@ -57,7 +57,7 @@
 /datum/quirk/blooddeficiency
 	name = "Blood Deficiency"
 	desc = "Your body can't produce enough blood to sustain itself."
-	icon = FA_ICON_TINT
+	icon = "tint"
```

Comment:
this is actively making the code worse, I don't think any of our quirk icons are really majorly different?